scripts/provisioning-tests

```diff
@@ -39,7 +39,12 @@ eval "$(grep '^ENV CATTLE_WINS_AGENT' package/Dockerfile | awk '{print "export "
 eval "$(grep '^ENV CATTLE_CSI_PROXY_AGENT' package/Dockerfile | awk '{print "export " $2 "=" $3}')"
 eval "$(grep '^ENV CATTLE_KDM_BRANCH' package/Dockerfile | awk '{print "export " $2 "=" $3}')"
 
-export SOME_K8S_VERSION=$(curl https://raw.githubusercontent.com/rancher/kontainer-driver-metadata/dev-v2.6/data/data.json | jq -r ".$DIST.channels[0].latest")
+if [ "$DIST" = "rke2" ]; then
+    export SOME_K8S_VERSION="v1.22.13+rke2r1"
```

Comment:
I do not think hardcoding this is a good idea.

Reply:
Rather than us working around this, we should either fix KDM or have the K3s/RKE2 team re-release the RKE2 version with correct assets.